### PR TITLE
Rok onboarding suggestions - small docs changes

### PR DESCRIPTION
**Pull request by ernopp**
Tested with improbadoc

Suggestions from 
https://docs.google.com/document/d/1eVbLtb9u_VFYDmsb8VjT2eTnsCMbe2aL82szTzHpfRM/edit#

### CODE REVIEW

File: docs/content/get-started/open-project.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Launch the Unity Editor. It should automatically detect the project but if it doesn't, select **Open** and then select `gdk-for-unity-fps-starter-project/workers/unity`.
 
+<%(Callout type="warning" message="
+It will take about 10 minutes for Unity to load the Starter Project for the first time. In the meantime, 
+why not take a stroll on our [Games Blog](https://improbable.io/games/blog)?" )%>
```
Comment (paulbalaji) [inline docs/content/get-started/open-project.md]: worth mentioning that subsequent loads/opens of the project will not take as long

File: docs/content/get-started/build-workers.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ As you will be launching a cloud deployment, you need to build out the code exec
 
 In the Unity Editor, you first need to make sure Burst compilation is **disabled** from **Jobs** > **Enable Burst Compilation**. Then you can build your workers from the SpatialOS menu by clicking **Build for cloud** > **All workers**.
 
-<%(Callout type="info" message="Building workers for the first time may take while. Why not make yourself a cup of tea while you wait?")%>
+<%(Callout type="info" message="Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea while you wait? Or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos)? Or both!")%>
 
 ![SpatialOS menu in Unity]({{assetRoot}}assets/unity-spatialos-menu.png)
 
-> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`
+> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. (You may want to only display Info and Error messages in the Console.)
```
Comment (paulbalaji) [inline docs/content/get-started/build-workers.md]: maybe rephrase to "want to hide Warnings in the Unity Console" ? I think it's fine either way - depends on what you think is clearer for a user

File: docs/content/get-started/upload-launch.md [hunk 3]
```diff
@@ -56,14 +56,13 @@ When launching a cloud deployment you must provide three parameters:
 * **the assembly name**, which identifies the worker assemblies to use. These are the assemblies you uploaded earlier so the name should match the one you passed to `spatial cloud upload`.
 * **a launch configuration**, which is a JSON file that declares the world and load balancing configuration.
 * **a name for your deployment**, which is used to label the deployment in the SpatialOS web Console. It consists of lower-case letters, digits, and `_` and can be up to 32 characters long.
+* **(optional) a region to deploy to**, which default to clusters located in the US. If you’re in Europe, add the `--cluster_region=eu` flag for lower latency.
```
Comment (paulbalaji) [inline docs/content/get-started/upload-launch.md]: defaults

File: docs/content/get-started/build-workers.md [hunk 4]
```diff
@@ -4,11 +4,11 @@ As you will be launching a cloud deployment, you need to build out the code exec
 
 In the Unity Editor, you first need to make sure Burst compilation is **disabled** from **Jobs** > **Enable Burst Compilation**. Then you can build your workers from the SpatialOS menu by clicking **Build for cloud** > **All workers**.
 
-<%(Callout type="info" message="Building workers for the first time may take while. Why not make yourself a cup of tea while you wait?")%>
+<%(Callout type="info" message="Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea while you wait? Or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos)? Or both!")%>
```
Comment (jessicafalk) [inline docs/content/get-started/build-workers.md]: should the end of the first sentence be: "may take **a** while"?

would also merge the two questions together: 
Why not make yourself a cup of tea or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos) while you wait?

Comment (ElleEss) [inline docs/content/get-started/build-workers.md]: We are trying to avoid covering all the docs in multi-coloured messages .
  ```suggestion
>Building workers for the first time may take while (about 10 minutes). Why not make yourself a cup of tea or check out [our Youtube channel](https://www.youtube.com/channel/UC7BE8B2yUeQxPvZytk47NYw/videos) while you wait?
```

Comment (ElleEss) [reply docs/content/get-started/build-workers.md]: +1

File: docs/content/get-started/build-workers.md (hunk 2 above)
Comment (ElleEss) [reply docs/content/get-started/build-workers.md]: This doesn't really explain why you want to. I am guessing because there's a lot of noise in the Console window?
If so, "You may want to..." would be better as....

```suggestion
> **It has finished building when:** You see the following message in the Unity Editor's Console window: `Completed build for Cloud target`. (As the build process displays lots of warnings in the Unity Console windows which you don't need to take action on, you could choose to set the Console window to display only info and error messages, so you can see the relevant messages.)
```

File: docs/content/get-started/open-project.md (hunk 1 above)
Comment (ElleEss) [inline docs/content/get-started/open-project.md]: There's too many colourful messages and so very many warning messages (whatever is the latest thing we notice), so better to keep it as a `>`.
Also, no need for predictions on the future with "it will take". Keep it present simple as a universal fact ("it takes"). It makes the text stronger.
We say "open" a Unity project in the Unity Editor, rather than "load" a project.
```suggestion
>The first time you open the Starter Project in your Unity Editor, It takes about 10 minutes; it's much quicker to open after this. (While you are waiting, you could look at our [Games Blog](https://improbable.io/games/blog).)
```

File: docs/content/get-started/open-project.md [hunk 5]
```diff
@@ -23,9 +27,11 @@ There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding
 1. Open the **Navigation** window (Unity Editor menu: **Windows** > **AI** > **Navigation**).
 1. Select the **Bake** tab, and then the **Bake** button.
 
-You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity displays the correct icon.
+You can verify that the NavMesh has been baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator**, and checking that Unity shows a Navmesh object (in most cases the logo will update to the below).
```
Comment (ElleEss) [inline docs/content/get-started/open-project.md]:
```suggestion
You can verify that the NavMesh is baked correctly by navigating to **Assets** > **Fps** > **Scenes** > **FPS-SimulatedPlayerCoordinator** and checking that there's a NavMesh object in the **Hierarchy** window. (The NavMesh object is usually represented by the icon below.)
```

A few bits and bobs here. The phrase "the below" is not so good English.  Also, it's worth saying where you can see the NavMesh object.

File: docs/content/get-started/open-project.md [hunk 6]
```diff
@@ -23,9 +27,11 @@ There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding
...
 <img src="{{assetRoot}}assets/navmesh-fixed.png" style="margin: 0 auto; display: block;" />
 
+<%(Callout type="warning" message="
```
Comment (ElleEss) [inline docs/content/get-started/open-project.md]:
```suggestion

```
Remove warning because it's a sea of red otherwise.

File: docs/content/get-started/open-project.md [hunk 7]
```diff
@@ -23,9 +27,11 @@ There is a [known issue]({{urlRoot}}/known-issues) in the Unity Editor regarding
...
+You may have a number of _warnings_ in the Unity editor console, which you can ignore at this stage. We recommend only showing info and error messages in the Console." )%>
```
Comment (ElleEss) [inline docs/content/get-started/open-project.md]:
```suggestion
> You may get a number of warnings displayed in the Unity Editor Console window. You can ignore the warnings at this stage, so you can use the message icons on the right-hand side of the Console window to set it to display only info and error messages.
```
Put as a simple `>`.
Warnings doesn't need to be italics.
Unity Editor Console window (note that's what it is called and all apart from "window" have upper case initials).

File: docs/content/get-started/upload-launch.md (hunk 3 above)
Comment (ElleEss) [inline docs/content/get-started/upload-launch.md]:
```suggestion
* **(optional) a region to deploy to**, which defaults to server clusters located in the USA. If you’re in Europe, add the `--cluster_region=eu` flag so the servers are in Europe. This gives you lower latency.
```

Comment (ElleEss) [reply docs/content/get-started/upload-launch.md]: This doesn't say what the "clusters" or that by using this flag you are deploying to servers in Europe.

File: docs/content/get-started/build-workers.md (hunk 4 above)
Comment (ElleEss) [reply docs/content/get-started/build-workers.md]: Note to self - Added in new PR.

File: docs/content/get-started/build-workers.md (hunk 2 above)
Comment (ElleEss) [reply docs/content/get-started/build-workers.md]: Note to self - added to new PR

File: docs/content/get-started/open-project.md (hunk 1 above)
Comment (ElleEss) [reply docs/content/get-started/open-project.md]: Note to self - added to new PR

File: docs/content/get-started/open-project.md (hunk 7 above)
Comment (ElleEss) [reply docs/content/get-started/open-project.md]: Note to self - added to new PR

File: docs/content/get-started/open-project.md (hunk 5 above)
Comment (ElleEss) [reply docs/content/get-started/open-project.md]: Note to self - added to new PR.

**Comment by jamiebrynes7** [comment]
Redone in #674. Closing this.
